source/fundamentals/connection/connection-options.txt

```diff
@@ -73,6 +73,28 @@ parameters of the connection URI to specify the behavior of the client.
      - Specifies whether to force dispatch **all** operations to the host
        specified in the connection URI.
 
+   * - **enableUtf8Validation**
+     - boolean
+     - ``true``
+     - 
+       Specifying ``true`` enables UTF-8 validation for the 
+       connection. MongoDB throws an error when 
+       it attempts to convert data that contains invalid 
+       UTF-8 characters. The validation adds validation processing 
+       overhead since it needs to check the data.
+
+       Specifying ``false`` disables UTF-8 validation for the 
+       connection. MongoDB does not throw errors when 
+       data contains invalid UTF-8 data. If you disable validation, 
+       your application avoids the validation processing overhead 
+       but cannot guarantee consistent presentation of invalid 
+       UTF-8 data. **Editing data** while validation is disabled
+       can result in loss of data. Disabling UTF-8 validation is a 
+       temporary workaround to query or export data only.
+
+       To learn more about UTF-8 characters, 
```

Comment:
Should we link to the utf-8 validation docs? https://www.mongodb.com/docs/drivers/node/current/fundamentals/bson/utf8-validation/

Reply:
I added a link to the closing sentence advising on this option.